test/evergreen/code_coverage/parallel_code_coverage.py

```diff
@@ -27,12 +27,44 @@
 # OTHER DEALINGS IN THE SOFTWARE.
 
 import argparse
+import re
 import json
 import logging
-import os
 import sys
-from code_coverage_utils import check_build_dirs, run_task_lists_in_parallel
-
+import subprocess
+import os
+from datetime import datetime
+from code_coverage_utils import check_build_dirs, run_task_lists_in_parallel, setup_build_dirs
+
+def run_task_list(task_list_info):
+    build_dir = task_list_info["build_dir"]
+    task_list = task_list_info["task_bucket"]
+    list_start_time = datetime.now()
+
+    env = os.environ.copy()
+    env["GCOV_PREFIX_STRIP"] = "4"
```

Comment:
Added a comment now, let me know what you think

Reply:
Looks good thanks!

If it's possible an `assert(env[CURRENT_GCOV_PATH].startswith("/data/mci/wiredtiger/build"))` would be nice to protect against this path changing under us in future, but I don't know if there's any `CURRENT_GCOV_PATH` env variable we can access. No worries if this isn't feasible.

Reply:
Unfortunately for evergreen builds, the wiredtiger portion in the example is filled with a randomly generated ID, so we can't assert it. So It can't be done